core/src/ldml/C7043_ldml.md

```diff
@@ -324,10 +324,115 @@ For each key:
+| 0 |  32  | ident      | `lyrs`                                   |
+| 4 |  32  | size       | int: Length of section                   |
+| 8 |  32  | count      | int: Number of layer lists               |
+|12 |  32  | layerstart | int: offset to first layer entry         |
+|16 |  32  | layercount | int: number of layer entries             |
+|20 |  96  | reserved   | padding                                  |
+
+Note layerstart must point after the end of the layer lists
+
+For each 'layers' - the layer lists
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | flags      | int: per-layers options                  |
+|24+|  32  | hardware   | str: layout (`us`,`iso`,`jis`,`abnt2`)   |
+|20+|  32  | target     | int: target vkey ID (0…255)              |
+|28+|  32  | layer      | int: index to first layer element        |
+|32+|  32  | layerCount | int: number of `layr` elements in list   |
+|36+|  96  | reserved   | short`[3]`: padding                      |
+
+- `flags`: a 32-bit bitfield defined as below:
+
+  | Bit position | Meaning  |  Description         |
+  |--------------|----------|----------------------|
+  |       0      | form     | 0: hardware          |
+  |       0      | form     | 1: touch             |
+
+For each layer element
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | id         | str: layer id such as `base` or `shift`  |
+|24+|  32  | modifier   | str: modifier string                     |
+|28+|  32  | row        | int: `rows` identifier of first row      |
+|32+|  32  | rowCount   | int: number of `rows` elements           |
+
+### C7043.2.x `rows`—Row list
+
+Each list is a row of keys in the `key2` element
+
+| ∆ | Bits | Name    | Description                              |
+|---|------|---------|------------------------------------------|
+| 0 |  32  | ident   | `rows`                                   |
+| 4 |  32  | size    | int: Length of section                   |
+| 8 |  32  | count   | int: Total number of row list elements   |
+|12 |  32  | reserved| padding                                  |
+
+> TODO-LDML: Should there be a '0-key2count terminator'?
+
+For each 'row' list element
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | key2       | int: index into `key2` element           |
+|24+|  32  | key2count  | int: count of `key2` elements            |
+
+### C7043.2.x `key2`—Secondary key list
+
+This is a list of vkeys, but ordered by rows.  The `rows` elements point into the start and length of entries here.
+
+> TODO-LDML: Should there be a '0-vkey terminator'?
+> And shold the 0th element also be a 0 vkey so that key2=0 means null?
```

Comment:
I think we don't need null term. But I think this should also be in the `lyrs` section.